Add workflow docs (#5493)
wso2/docs-is: en/includes/guides/user-self-service/configure-self-service-portal.md

    ``` bash
    {{my_account_org_link}}
    ```

> Listed below are self-service features that are available for the user in the **My Account portal**.

Reviewer: ```suggestion
Users can access the following self-service features from the **My Account portal**.
```
